README.md

```diff
@@ -1,8 +1,27 @@
 # SMART Health Cards
 
 ---
-### About
+## About
 
 This package will be used for the issuance of [SMART health cards](https://spec.smarthealth.cards/)
 
+## Progress Report (7.19.2022)
+- What's done: 
+  - Generating a JWS given a FHIR bundle, private/public key pair, JWK thumbprint for key ID
+  - Verifying the JWS using the given public key
+  - Verifying the JWS with the [smarth health card verifier portal](https://demo-portals.smarthealth.cards/VerifierPortal.html)
+  - Generating a scannable QR code from the generated JWS
+- What's incomplete:
+  - Generating QR code for large payloads. The smart health card docs indicate that cards are generally supposed to be very small payloads, however in cases where they exceed a certain threshold, they need to be split into chunks and each chunk would correspond to its own QR code. The walkthrough illustrates how the chunking process works.
+  - Organizing the issuer package code such that it can be used easily by other services. Currently, the issuer_test code lives alongside the issuer in the same package.
 
+## Local Development
+
+- Run `go mod vendor` to install dependencies
```

Comment:
If we're excluding the vendor directory from the repo, perhaps this should read `go mod tidy`.